docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[ts-management]]
+== Management tools
+
+This topic covers common troubleshooting issues when using the {elastic-sec} <<sec-manage-intro, management tools>>.
+
+[discrete]
+[[ts-endpoints]]
+=== Endpoints
+
+[discrete]
+[[ts-transform-failed]]
+.Required transform failed
+[%collapsible]
+====
+If you encounter a `“Required transform failed”` notice on the Endpoints page, you can usually resolve the issue by restarting the transform. See {ref}/transforms.html[Transforming data] for more information on transforms.
+
+[role="screenshot"]
+image::../images/endpoints-transform-failed.png[Endpoints page with Required transform failed notice]
+
+To restart a transform that’s not running:
+
+. Go to *Kibana* -> *Stack Management* -> *Data* -> *Transforms*.
+. Enter `endpoint.metadata_current-default` in the search box to find the transform for {endpoint-sec}.
+. Click the *Actions* icon (*...*) and do one of the following, depending on the value in the *Status* column:
+* `stopped`: Select *Start* to restart the transform. 
+* `failed`: Select *Stop* to first stop the transform, and then select *Start* to restart it.
++
+[role="screenshot"]
+image::../images/transforms-start.png[Transforms page with Start option selected]
+
+. Click *Start* in the confirmation message that displays to restart the transform.
```

Comment:
WDYT about putting the location first here? Typically, I try to aim for location before action - but in some instances the opposite works better. 

```suggestion
. On the confirmation message that displays, click *Start* to restart the transform.
```

Reply:
Agree. In this instance having the location first helps the flow of the overall sentence.